hash.c

```diff
@@ -4767,13 +4767,13 @@ env_delete_m(VALUE obj, VALUE name)
  *   ENV[name] -> value
  *
  * Returns the value for the environment variable +name+ if it exists:
- *   ENV['foo'] = 'bar'
- *   ENV['foo'] # => "bar"
+ *   ENV['foo'] = '0'
+ *   ENV['foo'] # => "0"
```

Comment:
Just for consistency, I'm suggesting that all examples use words as the names, and numbers (as strings) as the values.